image_writers_guide/sti_testing.adoc

```diff
@@ -10,64 +10,33 @@
 toc::[]
 
 == Overview
+As an STI builder image author, you can test your STI image locally and use the OpenShift build system for automated testing and continuous integration.
 
-This document covers how as a STI builder image author you can test your
-STI image locally and how you can use the OpenShift build system for automated
-testing and continuous integration.
+NOTE: See the link:sti.html[STI Requirements] topic to learn more about the STI architecture before reading further.
 
-Read more about the link:sti[STI architecture and requirements] before reading
-any further.
+As described in the link:sti.html[STI Requirements] topic, STI requires `assemble` and `run` scripts to be present in order to successfully execute the STI build. Providing the `save-artifacts` script reuses the build artifacts, and providing the `usage` script prints the usage to console when someone runs the Docker image outside of the STI.
 
-== The Goal
-
-As described in the STI documentation, STI requires 'assemble' and 'run'
-commands to be present in order to successfully execute the STI build.  You can
-also provide the 'save-artifacts' command for reusing the build artifacts and
-the 'usage' command that prints the usage to console when someone runs the
-Docker image outside the STI.
-
-The goal of testing the STI image is to make sure that all commands described
-above work properly, even if the base Docker image changed or the tooling that
-the commands are using was updated.
+The goal of testing an STI image is to make sure that all of these described commands work properly, even if the base Docker image has changed or the tooling that the commands use was updated.
```

Comment:
[Suggestion]
s/the tooling that the commands use was updated/the tooling used by the commands was updated